improve-file_type_blacklist-overwrite (#1660)
ezsystems/developer-documentation: docs/guide/images.md

First, enable adding SVG files to content by removing them from the blacklist 
of allowed MIME types.

> To do it, overwrite `ibexa.site_access.config.default.io.file_storage.file_type_blacklist` defined in `Core/Resources/config/default_settings.yml` so that `svg` is removed from the Blacklist. This can be done per Site Access using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).

Reviewer: ```suggestion
To do it, overwrite `ibexa.site_access.config.default.io.file_storage.file_type_blacklist` defined in `Core/Resources/config/default_settings.yml` so that `svg` is removed from the blacklist.
You can do it per Site Access by using [SiteAccess-aware configuration](multisite/siteaccess_aware_configuration.md).
```
